### DISCUSSION TITLE
Case Sensitivity Issue with RustDesk URL Scheme: ID Automatically Converted to Lowercase

### DISCUSSION THREAD

**Discussion by remoover** [rustdesk/rustdesk#10280] [General]
### Bug Description

When attempting to open RustDesk using the URL scheme **rustdesk://MMPCS-01,** the application converts the ID to lowercase (mmpcs-01), resulting in an ID not found error. The original ID, **MMPCS-01**, works correctly when entered manually.

### How to Reproduce

    1, Open a browser or a link handler that supports URL schemes.
    2, Enter the URL: rustdesk://MMPCS-01
    3, Observe that RustDesk converts the ID to lowercase and returns an "ID not found" error.

### Expected Behavior

The ID should remain in its original case (MMPCS-01) when opened via the URL, so it can be recognized correctly by RustDesk.

### Operating system(s) on local (controlling) side and remote (controlled) side

Win10, Win11

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.5

### Screenshots

![image](https://github.com/user-attachments/assets/2c77247c-e76e-4cd2-93b8-ead0e65cca43)

**Comment by xlionjuan** [comment]
I guess it is because URL **isn't** case sensitive.
